src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -901,8 +901,8 @@ protected virtual JwtSecurityToken ValidateSignature(string token, TokenValidati
                     return jwtToken;
             }
 
-            bool kidMatched = false;           
-            IEnumerable<SecurityKey> keys = null;
+            bool kidMatched = false;
+            IEnumerable<SecurityKey> keys = null;          
```

Comment:
please remove trailing space (it looks like it was there before :-) )

Reply:
done